sdk.conf

```diff
@@ -1 +1 @@
-osgeo4a_version=20200705
+osgeo4a_version=linux_build
```

Comment:
This will mean the latest version?

Reply:
We push:
 - from tags/releases on OSGeo4A an image with the tag
 - from branch pushes on OSGeo4A an image with the branch name (that's what happened here)

The next "release" will be available as soon as it's done

Reply:
```suggestion
osgeo4a_version=20200731
```

About to be released